README.md

```diff
@@ -29,6 +29,11 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 * [MTAR builder](https://www.npmjs.com/package/mbt) (`npm install -g mbt`)
 * [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html), Install cf-cli and run command `cf install-plugin multiapps`.
 
+> [!Note]
+> Reading attachment behaviour depends on the version of cds-services version which is used by the CAP application.
```

Comment:
You can rephrase this to:
The behaviour of clicking attachment and opening it varies based on the version of cds-services used by the CAP application.

Reply:
Changed